openreview/venue/invitation.py

```diff
@@ -328,10 +342,13 @@ def set_review_invitation(self):
             signatures=[venue_id],
             cdate=review_cdate,
             duedate=review_duedate,
-            expdate = review_expdate,
+            expdate = review_expdate,            
             date_processes=[{ 
                 'dates': ["#{4/cdate}"],
-                'script': self.cdate_invitation_process              
+                'script': self.invitation_edit_process              
+            }, { 
+                'dates': [self.update_date_string],
+                'script': self.invitation_edit_process             
             }],
```

Comment:
Not a necessary change, but you can achieve the same like this:

```python
date_processes=[{
    'dates': ["#{4/cdate}", self.update_date_string],
    'script': self.invitation_edit_process
}]
```

Reply:
ohhhhh, you are right. Let me change it.